src/main/java/org/zone/region/flag/entity/player/display/bossbar/BossBarMessageDisplayType.java

```diff
@@ -33,7 +33,7 @@
         if (displayTypeID == null) {
             throw new IOException("Couldn't get the display type");
         }
-        return new BossBarMessageDisplay(0, BossBar.Color.WHITE, BossBar.Overlay.PROGRESS);
+        return new BossBarMessageDisplayBuilder().setProgress(0).setColor(null).setOverlay(null).build();
```

Comment:
The joy of a builder is your dont need to specify ``setProgress(0)`` or ``setColor(null)`` if they are the default value. This can just be ``new BossBarMessageDisplayBuilder().build()``

Reply:
mm